portfolio/src/main/webapp/script.js

```diff
@@ -12,14 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
+/** Retrieves data from the /data page and displays it. */
+function getComments() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    json.forEach(commentText => createPElement(commentText));
+  });
+}
+
+/** Creates p elements and sets the text to comments pulled from \data page*/
```

Comment:
nit: Let's add a period at end of statement and trailing space before close comment

Reply:
Done.